test/extended/util/client.go

```diff
@@ -68,6 +68,7 @@ import (
 	mcv1client "github.com/openshift/client-go/machineconfiguration/clientset/versioned"
 	oauthv1client "github.com/openshift/client-go/oauth/clientset/versioned"
 	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned"
+	dnsrecordclientset "github.com/openshift/client-go/operatoringress/clientset/versioned"
```

Comment:
Maybe `ingressv1client` is better ? you could check other lines and see how they are named.